docs/running-on-kubernetes.md

```diff
@@ -1356,6 +1356,15 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>3.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.job.queue</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    The name of the queue to which the job is submitted. This info will be stored in configuration
+    and passed to specified feature step (such as `VolcanoFeatureStep`).
```

Comment:
Please remove
```
 (such as `VolcanoFeatureStep`)
```

Reply:
Also, `specified feature step` is ambiguous if this configuration is progagated to all feature steps.